### ISSUE TITLE
False positives for `no-unused-vars` on gts files

### ISSUE THREAD

**Issue by gitKrystan** [ember-cli/eslint-plugin-ember#1666]
```
import { on } from '@ember/modifier';
import Component from '@glimmer/component';

export interface MyComponentSignature {
  Args: {
    // error  'e' is defined but never used  no-unused-vars
    onClick: (e: Event) => void;
  };
}

export default class MyComponent extends Component<MyComponentSignature> {
  <template>
    <div {{on "click" @onClick}}>
      Click Me
    </div>
  </template>
}
```

**Comment by gitKrystan** [comment]
Closing bc I think this was a config issue

**Comment by charlesfries** [comment]
@gitKrystan Mind sharing what you changed in your config to fix this issue?

**Comment by gitKrystan** [comment]
I wish I remembered! ❤️

**Comment by NullVoxPopuli** [comment]
for gts, don't we want to use https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-unused-vars.md

instead of the eslint native one?
